charts/jenkins/Chart.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
```

Comment:
shouldn't the README be updated too? isn't there instructions for helm 2 there?

Reply:
good point. I updated this and removed helm v2 from README.